index.html

```diff
@@ -1,20 +0,0 @@
-<!DOCTYPE html>
-
-<html lang="en">
-    <head>
-        
-    </head>
-    <body>
-        <link rel="stylesheet" href="styles.css">
-        <canvas id="canvas" width="100" height="100"></canvas>
-        <script type="text/javascript" src="index.js"></script>
-        <script src="https://cdn.jsdelivr.net/npm/p5@1.6.0/lib/p5.js"></script>
-
-        <script src="path/to/p5.sound.js"></script>
```

Comment:
this path is still broken it seems like. It should link to the CDN version on cdn.jsdelivr.net or the local copy if you have downloaded the file and included it in the repo.